sklego/meta/zero_inflated_regressor.py

```diff
@@ -108,7 +108,7 @@ def fit(self, X, y, sample_weight=None):
                 logging.warning("Classifier ignores sample_weight.")
                 self.classifier_.fit(X, y != 0)
```

Comment:
Just as a side note after taking a look at the internals: as `classifier_` is supposed to classify zeros, wouldn't make sense to  fit on `y==0`? If so, that's a total of 3 lines that need changes. Not big deal anyway